workflow/rules/assembly.smk

```diff
@@ -41,7 +41,7 @@ rule assembly_megahit:
         " > {log} 2>&1"
 
 
```

Comment:
I was thinking of separating the assembly for ONT and illumina. Are they sharing most of the steps?

Reply:
Actually, they are already separated. One just has to expand the config and probably this function (https://github.com/IKIM-Essen/uncovar/pull/305/files#diff-973dfb502bde3e2836e467b210d176f7b91c2b1b0dd8adde0628827af2527371R181) to make the ONT assembler switchable. I also used the usuall contig paths to ensure, that we can expand the assembler comparison.